trace-resources/trace-resources.cabal

```diff
@@ -0,0 +1,76 @@
+cabal-version:        2.4
+name:                 trace-resources
+version:              0.1.0.0
+
+synopsis:             Package for tracing resources for linux, mac and windows
+author:               Juergen Nicklisch-Franken
+maintainer:           operations@iohk.io
+copyright:            2021 IOHK
+extra-source-files:   CHANGELOG.md
+                      README.md
+                      src/Cardano/Logging/Resources/os-support-darwin.h
+                      src/Cardano/Logging/Resources/os-support-win.h
+
+library
+  hs-source-dirs:     src
+  exposed-modules:    Cardano.Logging.Resources
+                      Cardano.Logging.Resources.Types
+
+  default-language:   Haskell2010
+  default-extensions: OverloadedStrings
+  build-depends:      base >=4.12 && <5
+                      , trace-dispatcher
+                      , text
+                      , aeson
+
+  if os(windows)
+     build-depends:    Win32
+  else
+     build-depends:    unix
+
+  ghc-options:         -Wall
+                       -Wcompat
+                       -Wincomplete-uni-patterns
+                       -Wincomplete-record-updates
+                       -Wpartial-fields
+                       -Widentities
+                       -Wredundant-constraints
+                       -Wmissing-export-lists
+                       -Wno-incomplete-patterns
+
+  if os(linux)
+    exposed-modules:   Cardano.Logging.Resources.Linux
+  if os(windows)
+    exposed-modules:   Cardano.Logging.Resources.Windows
+    c-sources:         src/Cardano/Logging/Resources/os-support-win.c
+    include-dirs:      src/Cardano/Logging/Resources/
```

Comment:
It's more common to use `cbits` and `include` directories for `*.c` and header files.

Reply:
See above

Reply:
Yes, cardano-crypto-praos uses the cbits directory, but no include.

Reply:
Done